### ISSUE TITLE
Date in title is always one day off

### ISSUE THREAD

**Issue by huyvohcmc** [thien-do/memos-pub#21]
Looking at the code:
https://github.com/thien-do/memos-pub/blob/514de8d64339c41690986f57ed2a8b6d363eaa0b/lib/blog/dir/entry.tsx#L16
```
d = new Date("03.22.2022");
> Tue Mar 22 2022 00:00:00 GMT+0700 (Indochina Time)
d.toISOString()
> '2022-03-21T17:00:00.000Z'

**Comment by thien-do** [comment]
Hi @huyvohcmc ! Thanks for the report. We fixed the issue.

However, a small problem is that I checked your blog (100 thanks for using memos.pub!) and unfortunately we can only support ISO format by default. This means the list at https://huyvohcmc.memos.pub/notes isn't sorted correctly, because we don't recognize the date.

**Why**

Previously we used a loose parsing (`new Date(...)`). This accepts a wide range of format, but we can't explicitly force a timezone, and more importantly we can't differentiate between dd/mm and mm/dd (e.g. 03/04/2022 can be both April or March).

So in this fix we only support ISO format, and leaving custom date format into the config file.

**What should be done**

Option 1: I can have a PR in your repo to change the date (in the file names) to ISO format (e.g. 12.24.2022 to 2022-24-12)

Option 2: We will have support for a config file, which lets you choose a custom format for parsing date. I'm not sure when will we have this
